spec/features/users/sign_up_spec.rb

```diff
@@ -163,6 +163,42 @@
     expect(page).to have_current_path account_path
   end
 
+  it 'does not allow a user to choose piv/cac as 2FA method during sign up' do
```

Comment:
Should we also add a controller test to make sure a user can't visit the `piv_cac` endpoint directly when `piv_cac_available_for_agency?` is `false`?

Reply:
I thought about it, but I'm not too worried. If someone does manage to get their piv/cac to work, it's okay. The main reason for showing only when they're associated with a supported agency is to avoid confusion for the majority of government employees we don't support yet.